test/run.py

```diff
@@ -255,6 +261,7 @@ def register():
     test_generic(name='ocaml', test_file='test.ml', expected='Hello, world!\n', with_head=True)
     test_generic(name='go', test_file='test.go', expected='hello\n', with_head=True)
     test_generic(name='sbcl', test_file='test.lisp', expected='hello', with_head=True)
+    test_pony_head()
```

Comment:
Add a test for `pony-0.11.3`.
Maybe, you can use `test_generic` function: `test_generic(name='pony', test_file='test/main.pony', excepted='hello\n', with_head=True)`.